networks.json

```diff
@@ -140,13 +140,13 @@
         "links": [
             {
                 "name": "Mainnet",
-                "chain_id": "likecoin-chain-sheungwan",
+                "chain_id": "likecoin-mainnet-2",
                 "url": "https://likecoin.bigdipper.live"
             },
             {
                 "name": "Testnet",
-                "chain_id": "likechain-testnet-taipei-1",
-                "url": "https://taipei.likecoin.bigdipper.live"
+                "chain_id": "likecoin-public-testnet-3",
+                "url": "https://testnet.likecoin.bigdipper.live"
```

Comment:
Is it currently working?

Reply:
No, waiting for @rav-11 to point it then we'll merge

Reply:
It's working now, we can merge